ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/byo-kms/etcd-encryption/deprovision/cucushift-installer-rehearse-aws-ipi-byo-kms-etcd-encryption-deprovision-chain.yaml

```diff
@@ -0,0 +1,8 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-byo-kms-etcd-encryption-deprovision
+  steps:
+  - chain: cucushift-installer-rehearse-aws-ipi-deprovision
+  - ref: aws-deprovision-stacks
```

Comment:
We do not need this step any more, right?

Reply:
fixed.